images/bootstrap/create_bazel_cache_rcs.sh

```diff
@@ -18,35 +18,41 @@ CACHE_PORT="8080"
 
 # get the installed version of a debian package
 package_to_version () {
-    dpkg-query --showformat='${Version}' --show $1
+    dpkg-query --showformat='${Version}' --show "$1"
 }
 
 # look up a binary with which and return the debian package it belongs to
 command_to_package () {
-    local binary_path=$(readlink -f $(which $1))
+    local binary_path
+    binary_path=$(readlink -f "$(which "$1")")
```

Comment:
humm this is from shell lint?

Reply:
yeah, I don't like this new style, unless you also go and move all of the local definitions to the top of the function, c-style.

Reply:
yes, if you don't do assignment from subshells separately from local / export it masks the exit code https://github.com/koalaman/shellcheck/wiki/SC2155

Reply:
TIL. bash is the worst.

Reply:
I'm not a fan of this style either, but the shellcheck lint seems convincing. I'm not sure the c style would be better, espcially since this is only relevant to assignment which uses subshells, not literals / string formation